.travis.yml

```diff
@@ -2,19 +2,29 @@ sudo: required
 dist: trusty
 language: shell
 services:
-  - docker
```

Comment:
Why are these no longer indented? [It's the default YAML syntax](http://yaml.org/spec/1.2/spec.html#id2759963).

Reply:
It was regenerated to use Travis' generator

Reply:
Fwiw prettier now supports yaml (https://prettier.io/blog/2018/07/29/1.14.0.html#yaml)... :-)

Reply:
I run `travis lint` before I commit changes to `.travis.yml` and it does flatten the structure as well